### PR TITLE
a70q: Add 20231202 build!

### PR DESCRIPTION
**Pull request by Haky86**

### CODE REVIEW

File: changelog_a70q.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Highlights & Device Specific Changes:
```
Comment (ghost) [inline changelog_a70q.txt]: Bad filename, must be "a70q_changelog.txt"

**Review by ghost — CHANGES_REQUESTED** [review]

File: changelog_a70q.txt (hunk 1 above)
Comment (Haky86) [reply changelog_a70q.txt]: that's what I got in the generated build, also 13.x was the same https://github.com/crdroidandroid/android_vendor_crDroidOTA/blob/13.0/changelog_a70q.txt unless it changes in 14.x, thanks for notify!

Comment (Haky86) [reply changelog_a70q.txt]: @tw0xa I updated shall I close the PR and add new one?

Comment (ghost) [reply changelog_a70q.txt]: No need to open a new one, just correct filename and push it to this pr :)

Comment (Haky86) [reply changelog_a70q.txt]: @tw0xa done :)

**Review by ghost — APPROVED** [review]

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**